.github/workflows/codeql.yml

```diff
@@ -0,0 +1,54 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["main"]
+
+jobs:
+  analyze:
+    name: Analyze (${{ matrix.language }})
+    runs-on: "ubuntu-latest"
+    timeout-minutes: 360
```

Comment:
CodeQL only takes about 2 minutes, is there a reason we put a timeout here and chose 360?

Reply:
No need to change it if it matches the existing code for mongodb-client-encryption!